packages/core/src/components/Timeline/styles.ts

```diff
@@ -0,0 +1,28 @@
+import { StyleSheet } from '../../hooks/useStyles';
+
+export const styleSheetItem: StyleSheet = ({ color, unit }) => ({
+  wrapper: {
+    marginLeft: unit * 0.5 - 1,
+    borderLeft: `2px solid ${color.core.neutral[3]}`,
+    paddingLeft: unit * 2.5,
+    paddingBottom: unit * 7,
+  },
+
+  wrapper_oldest: {
+    paddingLeft: unit * 3,
+  },
+});
+
+export const styleSheetSimpleItem: StyleSheet = ({ color, unit }) => ({
+  wrapper: {
+    marginLeft: unit * 0.5 - 1,
+    borderLeft: `2px solid ${color.core.neutral[3]}`,
+    paddingLeft: unit * 2.5,
+    height: unit * 2.75,
```

Comment:
is this heigh restriction necessary? both `wrapper` styles are pretty similar except for the padding bottom. if consolidated with paddingBottom, is the height still needed?

Reply:
I'll see if I can remove but I think it will be required since the "secondary" item style actually has no `children` prop - so the height of the wrapper here is actually just providing the default "empty space" below the secondary item's title

Reply:
Looks like I was able to clean it up some, good suggestion!